### ISSUE TITLE
Very annoying bug in Pokémon Emerald when holding "B", "Right" and "Fast forward (held)" button at same time...

### ISSUE THREAD

**Issue by MaxWhite3790** [mgba-emu/mgba#2313]
When I try to run to right by holding the "B" button and the "Right" button while holding the "Fast forward (held)" button, all at the same time, the character just doesn't run... He just walks...

What is even more weird, is that the same thing doesn't happens if I try to run to left (Doing all the same thing)...

**Comment by dracc** [comment]
On what platform?
Assuming PC makes it sound to me like you're using a cheap keyboard.

**Comment by MaxWhite3790** [comment]
> On what platform?
> Assuming PC makes it sound to me like you're using a cheaply made keyboard.

PC, yes... The keyboard is the problem?

**Comment by dracc** [comment]
> PC, yes... The keyboard is the problem?

That's what I'm thinking. It's not unusual for cheaply made keyboards to not allow more than 2 or 3 keys at once, depending on where the keys are located.

**Comment by endrift** [comment]
Yeah this definitely sounds like a keyboard issue. Key blocking is a real problem with keyboards. Try a controller if you have one.

**Comment by Yave-Yu** [comment]
So common keyboard conflict issue.
If you are using membrane keyboard, it's normal, try to use mechanical keyboard.

**Comment by endrift** [comment]
If looking for a new keyboard the magic term to look for is NKRO or "key rollover", which refers to how many keys you can typically press without blocking.

**Comment by RunisDauphin** [comment]
I know why this happen. Several keyboards (for example, my laptop one) do not allow the use of three-key combinations simultaneously. They are certain combinations, not all of them are forbidden, and depends on the keyboard.

The probleme will affect any programme you use, not only this emulator. Your solution is change keyboard.
